src/extension-consumer/src/topic/mod.rs

```diff
@@ -64,6 +79,11 @@ impl TopicCmd {
             TopicCmd::List(list) => {
                 list.process(out, fluvio).await?;
             }
+            TopicCmd::Metadata => {
+                use crate::common::t_println;
+                t_println!(out, "{}", format!("metadata: {:#?}", self.metadata()));
```

Comment:
When I run this I get an error `Connection refused (os error 111)`

Reply:
that's weird.

Reply:
Yeah... this is what I get:
```
simlay@tower-1:~/projects/infinyon/fluvio (extension-command-protocol)$ cargo run --bin fluvio -- topic metadata
   Compiling fluvio-extension-consumer v0.1.0 (/home/simlay/projects/infinyon/fluvio/src/extension-consumer)
   Compiling fluvio-cli v0.4.0 (/home/simlay/projects/infinyon/fluvio/src/cli)
    Finished dev [unoptimized + debuginfo] target(s) in 10.26s
     Running `target/debug/fluvio topic metadata`
Error: 
   0: Fluvio client error
   1: Fluvio socket error
   2: Connection refused (os error 111)

Location:
   src/cli/src/bin/main.rs:12

Backtrace omitted.
Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

Reply:
I removed this because it's broken and I don't see the need for a statically bound command to need a `metadata` command.